src/start_server.ts

```diff
@@ -456,6 +468,23 @@ async function startWithFirstAvailablePort(
 async function tryStartWithPort(
     options: ServerOptions, app: express.Application, port: number) {
   const server = await createServer(app, options);
+
+  if (options.liveReloadPath) {
+    const wss = new WebSocketServer({server: server});
+    wss.on('connection', (ws: WebSocket) => {
+      const watcher = watchFile(options.liveReloadPath);
```

Comment:
If `liveReloadPath` points to a directory it would be nice to recursively watch that directory and its children. I suspect that watchFile when given a directory will see changes to that directory (in some filesystems at least) but not changes inside subdirectories.